packages/code-du-travail-frontend/src/sources.js

```diff
@@ -33,6 +33,6 @@ export const getRouteBySource = src => routeBySource[src];
 
 const sources = Object.keys(routeBySource);
 export const getExcludeSources = value => {
-  if (Boolean(value) === false) return [];
+  if (Boolean(value) === false) return "";
```

Comment:
This function now returns an `array` or a `string` ?

Reply:
it should aways returns a string, returning a string or Array is not a good idea